backend/app/api/views/course.py

```diff
@@ -0,0 +1,137 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.views import APIView
+from rest_framework.decorators import api_view, permission_classes
+
+
+from app.decorators.decorators import allowed_roles
+from app.models.course import Course
+from app.serializers.course import CourseNameSerializer
+
+
+class CourseView(APIView):
+    permission_classes = (IsAuthenticated,)
+
+    @allowed_roles(["student", "creator", "staff"])
+    def get(self, request, id=None, format=None):
```

Comment:
1. check id from the request path is valid
2. find the course by id or raise an exception (not found)
3. return the list of users in this course